cache/util/createCache.js

```diff
@@ -0,0 +1,21 @@
+const Cache = require('../Cache')
+
+module.exports = async function createCache (nodes, options) {
+  const cache = new Cache(nodes, options)
```

Comment:
Maybe I'm missing something, but your method signature in `Cache` suggests that there is only one parameter. 

```js
class Cache extends EventEmitter {
  constructor ({ logger, nodes, defaultTtl, redisClientOptions }) {}
}
```